servers/xr_server.cpp

```diff
@@ -352,6 +362,52 @@ PackedStringArray XRServer::get_suggested_pose_names(const StringName &p_tracker
 	return arr;
 }
 
+void XRServer::add_face_tracker(const StringName &p_path, Ref<XRFaceTracker> p_face_tracker) {
+	ERR_FAIL_COND(p_face_tracker.is_null());
+
+	if (!face_trackers.has(p_path)) {
+		// We don't have a tracker with this name, we're going to add it.
+		face_trackers[p_path] = p_face_tracker;
+		emit_signal(SNAME("face_tracker_added"), p_path, p_face_tracker);
+	} else if (face_trackers[p_path] != p_face_tracker) {
+		// We already have a tracker with this name, we're going to replace it.
+		face_trackers[p_path] = p_face_tracker;
+		emit_signal(SNAME("face_tracker_updated"), p_path, p_face_tracker);
+	}
+}
+
+void XRServer::remove_face_tracker(const StringName &p_path) {
+	// Skip if no provider is found
+	if (!face_trackers.has(p_path)) {
+		return;
+	}
+
+	// we send the signal right before removing it
```

Comment:
```suggestion
	// We send the signal right before removing it.
```

Reply:
Done - merged with the next comment to make for a better sentence. Now reads "// Send the removed signal, then remove the face tracker."